### ISSUE TITLE
Non-subscription/app/SaaS usage of Azure

### ISSUE THREAD

**Issue by ActionAnalytics** [bcgov/cloud-pathfinder#1806]
**Describe the issue**
Some Azure requests come to us for simple pieces of infrastructure.  They just pass through data like ADF, Data Factory, Data Lake, Databases, Machine Learning, Log storage but they need to come to us as they need a subscription with it.  Next 5 to 10 years statistic things going to SaaS in MS World, they need a hook as a charging unit to bill against and that hook is the subscription.  So we need to be involved.

**Additional context**
- AG Lifecycle management service came up, request to ADMS group to deal with.  Required Global Admin level access, but ADMS decided to do centrally.
- We're not supposed to be dealing with Microsoft until the contract is unblocked
- Definition of SaaS in Cloud Pathfinder is outside of cloud, the "SaaS" bits within cloud, we define as Tools within each cloud platform

**Definition of done**
- 1 hour meeting where Jon shows the nature of the current problem, shares proposed solutions, and team discusses with respect to SEA fit (or cutover)
- In the discussion will be Jon, Max, Warren, Julian, Bahaa
- List of questions we need to know before we speak to other groups
- Common set of vocab/common language to clarify within our group and outside 
- 30 minute meeting to refine tickets for next stages (assuming work drops out of this) and to coordinate with external resources (like Jesse's team that has key permissions needed) if needed.

**Comment by jon-mc-git** [comment]
Our example project for this is not coming online yet by the AG folks, as they are connecting the involved Microsoft Application Lifecycle Management cloud service (LCS) to on-prem only now, which does not require the Subscription-billing link yet.  Putting on hold for now until word back from AG

**Comment by wrnu** [comment]
will be tracked in a different ticket
